src/mqttcm_webcfg.c

```diff
@@ -120,7 +122,90 @@ void sendRbusEventWebcfgOnPublish(int mid)
 			MqttCMError("provider: rbusEvent_Publish onpublish event failed: %d\n", rc);
 	}
 }
+
+void sendRbusErrorto_mqtt(rbusError_t rc, char *topic_name)
```

Comment:
change this to "sendRbusErrorToMqtt" inorder to follow the same kind of function name we use in this repo

Reply:
sendRbusErrorto_mqtt function name changed to sendRbusErrorToMqtt